docs/deployment/index.md

```diff
@@ -17,14 +17,23 @@ _For more information about advanced configuration, read our documentation [here
 ```
 
 # Kubernetes
-We currently provide [a sample declaration](https://github.com/tomkerkhove/promitor/tree/master/deploy) which deploys all the required infrastructure on your Kubernetes cluster.
+We currently provide [a helm chart](https://github.com/tomkerkhove/promitor/tree/master/charts) which deploys all the required infrastructure on your Kubernetes cluster.
 
-Once downloaded, you can deploy it by running this command:
+You can then edit the values.yaml, or take a look at [this example yaml](https://github.com/tomkerkhove/promitor/blob/master/charts/local-values.yaml.example) to base your own on.
+
+You will need to provide the following values to minimally run the chart:
+- Application Id
+- Application Key
+- Tenant Id
+- Subscription Id
+- Resource Group
+- Metric Configuration
+
+After filling out the required fields, you can then deploy the chart by running this command:
```

Comment:
```suggestion
Once configured, you can easily deploy the Helm chart by running this command:
```

Reply:
Again, made slightly different changes for overall clarity, let me know if you want more changes